cmd/helm/install.go

```diff
@@ -285,12 +286,32 @@ func (v *values) Set(data string) error {
 	return nil
 }
 
+func typedVal(val string) interface{} {
+	if val == "true" {
+		return true
+	}
+
+	if val == "false" {
+		return false
+	}
+
+	if iv, err := strconv.ParseInt(val, 10, 64); err == nil {
```

Comment:
@technosophos I think the issue was more that people were using the quote function for integers. As far as I know, a simple `{{.Values.replicas}}` won't add quotes if it's a string.